store/gcworker/gc_worker.go

```diff
@@ -317,6 +396,15 @@ func (w *GCWorker) leaderTick(ctx context.Context) error {
 		return errors.Trace(err)
 	}
 
+	// Do keyspace delete range
+	if w.store.GetCodec().GetKeyspace() != nil {
+		err = w.runKeyspaceGCJob(ctx, concurrency)
```

Comment:
Is the *delete range owner* checked?

Reply:
It already in function `leaderTick` , so there is no need to check whether it's delete range owner?

Reply:
I got it. `checkLeader` will be run in a key-space scope and key-space specific GC states will be used instead of the global ones, therefore `checkLeader` will elect a GC leader in every key-space. I think it also needs explaination in comments